src/components/templates/PartyMap/components/RoomAttendance.tsx

```diff
@@ -1,50 +1,60 @@
-import React, { FC, useMemo } from "react";
+import React, { useMemo } from "react";
 
 import { usePartygoers } from "hooks/users";
 
-import { PartyMapRoomData } from "types/PartyMapRoomData";
-import { PartyMapVenue } from "types/PartyMapVenue";
+import { Room } from "types/rooms";
+import { PartyMapVenue } from "types/venues";
 
 import "./RoomAttendance.scss";
 
-interface PropsType {
+interface RoomAttendanceProps {
   venue: PartyMapVenue;
-  room: PartyMapRoomData;
+  room: Room;
 }
 
 const MAX_AVATARS_VISIBLE = 2;
 
-export const RoomAttendance: FC<PropsType> = ({ venue, room }) => {
+export const RoomAttendance: React.FC<RoomAttendanceProps> = ({
+  venue,
+  room,
+}) => {
   const partygoers = usePartygoers();
+
   const usersInRoom = useMemo(
     () =>
-      partygoers?.filter(
+      partygoers.filter(
         (partygoer) => partygoer.lastSeenIn?.[`${venue.name}/${room.title}`]
       ),
     [partygoers, venue.name, room.title]
   );
 
-  const numberOfUsersInRoom = usersInRoom?.length;
-  if (numberOfUsersInRoom < 1) return <></>;
```

Comment:
I think the logic was changed here. Was it tested?

Reply:
(as a general side note, it’s far easier to see what you are referring to if you select all of the relevant lines. In this case, all I can see is the deleted lines, and so need to go and open the file/find this section/etc to be able to comment further)

Reply:
Previously it would just render nothing at all when there are no users in the room. Now it will still render the ‘container div’, but just won’t have anything in it.

I believe I did visually test this and it seemed to look fine (though for reasons that I believe are outside of the scope of this PR, I don’t think I was ever seeing any users in this room list, which I assume has/will be fixed with your users related PR, and/or will make more sense to track down/solve as a new PR once this + your users PR are merged)